packages/client/src/apis/market/types.ts

```diff
@@ -6,51 +6,52 @@ import {
   EngineExecutePlaceOrderParams,
   EngineOrderParams,
 } from '@vertex-protocol/engine-client';
-import { OptionalSignatureParams, OptionalSubaccountOwner } from '../types';
 import {
-  QueryListTriggerOrdersParams,
   TriggerExecuteCancelOrdersParams,
   TriggerExecuteCancelProductOrdersParams,
   TriggerExecutePlaceOrderParams,
+  TriggerQueryListTriggerOrdersParams,
 } from '@vertex-protocol/trigger-client';
+import { OptionalSignatureParams, OptionalSubaccountOwner } from '../types';
 
 type ClientOrderParams<T> = Omit<OptionalSignatureParams<T>, 'order'> & {
   order: OptionalSubaccountOwner<EngineOrderParams>;
 };
 
-export type PlaceOrderParams = ClientOrderParams<EngineExecutePlaceOrderParams>;
+export type ExecutePlaceOrderParams =
+  ClientOrderParams<EngineExecutePlaceOrderParams>;
 
-export type CancelOrdersParams = OptionalSignatureParams<
+export type ExecuteCancelOrdersParams = OptionalSignatureParams<
   OptionalSubaccountOwner<EngineExecuteCancelOrdersParams>
 >;
 
-export type CancelProductOrdersParams = OptionalSignatureParams<
+export type ExecuteCancelProductOrdersParams = OptionalSignatureParams<
   OptionalSubaccountOwner<EngineExecuteCancelProductOrdersParams>
 >;
 
-export interface CancelAndPlaceOrderParams {
-  placeOrder: PlaceOrderParams;
-  cancelOrders: CancelOrdersParams;
+export interface ExecuteCancelAndPlaceOrderParams {
+  placeOrder: ExecutePlaceOrderParams;
+  cancelOrders: ExecuteCancelOrdersParams;
 }
 
-export type PlaceTriggerOrderParams =
+export type ExecutePlaceTriggerOrderParams =
   ClientOrderParams<TriggerExecutePlaceOrderParams>;
 
-export type CancelTriggerOrdersParams = OptionalSignatureParams<
+export type ExecuteCancelTriggerOrdersParams = OptionalSignatureParams<
   OptionalSubaccountOwner<TriggerExecuteCancelOrdersParams>
 >;
 
-export type CancelTriggerProductOrdersParams = OptionalSignatureParams<
+export type ExecuteCancelTriggerProductOrdersParams = OptionalSignatureParams<
   OptionalSubaccountOwner<TriggerExecuteCancelProductOrdersParams>
 >;
 
-export type MintLpParams = OptionalSignatureParams<
+export type ExecuteMintLpParams = OptionalSignatureParams<
   OptionalSubaccountOwner<EngineExecuteMintLpParams>
 >;
 
-export type BurnLpParams = OptionalSignatureParams<
+export type ExecuteBurnLpParams = OptionalSignatureParams<
   OptionalSubaccountOwner<EngineExecuteBurnLpParams>
 >;
 
-export type GetTriggerOrdersParams =
-  OptionalSignatureParams<QueryListTriggerOrdersParams>;
+export type QueryListTriggerOrdersParams =
```

Comment:
re: `TriggerQueryListTriggerOrdersParams` -- think we should find a better name for this. why can't this just be `QueryTriggerOrdersParams`. in the backend we have the concept of `Signed` vs `Unsigned` stuff, maybe we could do something similar here if the diff is just an optional signature.

Reply:
`TriggerQueryListTriggerOrdersParams` - the `Trigger` prefix is to indicate that this is talking to the the trigger service. It's clear enough with this specific case but for example - `GetIndexerOrdersParams` without `Indexer` would be `GetOrdersParams`: would this be talking to the engine or the indexer?